express/css/custom.css

```
@@ -98,12 +92,15 @@ font-family: 'Questrial', sans-serif;
     display: block;
   }
   
-  /* Change color of dropdown links on hover */
-  .dropdown-content a:hover {background-color: #ddd}
-  
-  /* Show the dropdown menu (use JS to add this class to the .dropdown-content container when the user clicks on the dropdown button) */
-  .show {display:block;}
 
+/* Change color of dropdown links on hover */
+.dropdown-content a:hover {background-color: #ddd;}
```

I would try and stick to conventional indentation whenever possible, even though it adds lines of code, it makes it easier to read, which is really the key to code happiness
```suggestion
.dropdown-content a:hover {
  background-color: #ddd;
}
```